init

```diff
@@ -135,6 +135,14 @@ fi
 # develop on them -> pre-commit hook not required -> we can use latest
 # packages.
 if [[ "$ci" == "false" ]]; then
+  if python3 --version | grep -q '^Python 3\.12'; then
```

Comment:
You should check `[ -f /usr/lib/python$py_ver/EXTERNALLY-MANAGED ]` instead, where `py_ver` is parsed by `py_ver=$(python3 --version | cut -d " " -f 2 | cut -d "." -f 1,2)`